ocs_ci/helpers/helpers.py

```diff
@@ -4451,32 +4451,38 @@ def check_selinux_relabeling(pod_obj):
     logger.info(f"{key} is present in inspect logs of application pod running node")
 
 
-def verify_log_exist_in_pod_logs(
-    pod_name,
+def verify_log_exist_in_pods_logs(
+    pod_names,
     expected_log,
     container=None,
     namespace=config.ENV_DATA["cluster_namespace"],
     all_containers_flag=True,
+    since=None,
 ):
     """
-    Verify log exist in pod logs.
+    Verify log exist in pods logs.
 
     Args:
-        pod_name (str): Name of the pod
-        expected_log (str): the expected logs in "oc logs" command
+        pod_names (list): Name of the pod
+        expected_log:
```

Comment:
Why did you remove the `(str): the expected logs in "oc logs" command` from the docstring?

Reply:
fixed